package.json

```diff
@@ -154,6 +154,10 @@
         "title": "Go: Add Import",
         "description": "Add an import declaration"
       },
+      {
+        "command": "go.import.workspace",
+        "title": "Go: Add import to workspace"
```

Comment:
The name is not very self explanatory....
How about `go.add.package.workspace` and `Go: Add Package to Workspace` ?